src/test/java/org/jqassistant/plugin/jee/cdi/test/set/beans/scope/javax/JavaxTypeWithSessionScopedMethod.java

```diff
@@ -0,0 +1,14 @@
+package org.jqassistant.plugin.jee.cdi.test.set.beans.scope.javax;
+
+import javax.enterprise.context.SessionScoped;
+
+import org.jqassistant.plugin.jee.cdi.test.set.beans.scope.ProducedBean;
+
+public class JavaxTypeWithSessionScopedMethod {
+
+    @SessionScoped
+    ProducedBean produceBean() {
+        return new ProducedBean();
```

Comment:
Please verify if this is supported by CDI or if Produces is required in addition. If not then this class and according rules shall be removed.